src/Util/Util.js

```diff
@@ -566,25 +566,72 @@ const Util = {
   },
 
   /**
+   * Underscore.js 1.8.3. For more information,
```

Comment:
Why don't we use underscore everywhere instead? We copy the function and have different versions flying around. Let's switch to having underscore/throttle as a dependency